app/javascript/components/ui/tip/tip-styles.scss

```diff
@@ -5,4 +5,10 @@
   background-color: $tip;
   font-size: 12px;
   color: $slate;
+
+  &.tooltip-dark {
```

Comment:
For now this is fine, but lets create an icebox task in Pivotal to refector the tooltips and styles to remove this horrible dependancy.